packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -103,25 +88,25 @@ The values for a configuration object are:
 
```

Comment:
In the description for ClientID in the table above, if you want to be consistent, you should use "application integration" rather than "app integration".  

I guess Mobile apps are called apps rather than applications, so your call if you want to change the "app"s in the descriptions for RedirectURI and Scopes too.